specs-go/version.go

```diff
@@ -25,7 +25,7 @@ const (
 	VersionPatch = 0
 
 	// VersionDev indicates development branch. Releases will be empty string.
-	VersionDev = "-dev"
+	VersionDev = "-rc.3-dev"
```

Comment:
Actually, I just remembered -- using `-dev` this way results in suboptimal behaviour with SemVer. SemVer treats `1.0.0-dev` as being a pre-release of `1.0.0` -- meaning that the `-dev` version is seen as "older". Another issue is that `-dev` is not conceptually a pre-release.

In runc and umoci we use `+dev` because that matches the correct version order behaviour with rpm and most other tools (`1.0.1 > 1.0.0+dev > 1.0.0`), though under SemVer `1.0.0+dev == 1.0.0` (I don't think this is a problem for a specification though, but maybe that's just me).

Maybe we should use `+dev` as well?

Reply:
Ah, this is outlined in `RELEASES.md`. I will submit a PR to make the change for future releases.

Reply:
See #1050.